core/src/main/java/com/backblaze/b2/client/B2StorageClientWebifierImpl.java

```diff
@@ -420,6 +421,15 @@ public B2FileVersion getFileInfo(B2AccountAuthorization accountAuth,
                 B2FileVersion.class);
     }
 
+    @Override
+    public B2FileVersion getFileInfo(B2AccountAuthorization accountAuth, B2GetFileInfoByNameRequest request) throws B2Exception {
+        return webApiClient.postJsonReturnJson(
```

Comment:
have you been able to run this?  did it work?

Reply:
No, I have no chance to run it. I have no test account to test. And where is no available mock for "server" connection.